### ISSUE TITLE
inefficiency in ```quad.form()```

### ISSUE THREAD

**Issue by RobinHankin** [RobinHankin/emulator#3]
The idiom for ```quad.form()``` is inefficient:

```drop(cprod(cprod(M, x), x))``` takes two conjugates and it should require only one.

Fix:
```
 library(cmvnorm)
 library(emulator)
 M <- matrix(rcnorm(4),2,2)
 x <- rcnorm(2)
 quad.form(M,x)
[1] -0.350405-1.11485i
 crossprod(crossprod(M,Conj(x)),x)
                   [,1]
[1,] -0.350405-1.11485i
```

We need to check the other functions too for this sort of infelicity

**Comment by RobinHankin** [comment]
The problem is, it's very tempting to use ```cprod()``` and ```tcprod()``` instead of ```crossprod()``` and ```tcrossprod()``` when possible, and this certainly makes for nicer more elegant code.  But it's inefficient and I think that is more important.

**Comment by RobinHankin** [comment]
fixed in ae6366b
